core/db/migrate/20161006104057_add_amount_remaining_to_store_credit_events.rb

```diff
@@ -0,0 +1,38 @@
+class AddAmountRemainingToStoreCreditEvents < ActiveRecord::Migration[5.0]
+  def up
+    add_column :spree_store_credit_events, :amount_remaining, :decimal, precision: 8, scale: 2, default: nil, null: true
+
+    Spree::StoreCredit.all.each do |credit|
```

Comment:
We need to avoid referencing the `Spree::StoreCredit` model and the constants here.

Instead we can define our own "AddAmountRemainingToStoreCreditEvents::StoreCredit" model here and use that instead.